stable_baselines/common/policies.py

```diff
@@ -751,3 +786,23 @@ def register_policy(name, policy):
         raise ValueError("Error: the name {} is alreay registered for a different policy, will not override."
                          .format(name))
     _policy_registry[sub_class][name] = policy
+
+
+def create_dummy_action_mask(ac_space, num_samples):
```

Comment:
utility functions that are used throughout to create / reshape the action_mask vectors. Used here, and in various algorithm _train_steps